.changeset/khaki-scissors-look.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/graphql-client': minor
+---
+
+Adds support for signal request option
```

Comment:
```suggestion
Adds support for `signal` request option
```

Reply:
Oh, didn't know we have markdown formatting in changesets...will fix